app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::DoctorsController < ApplicationController
+  def index
+    @doctors = Doctor.all.sample
```

Comment:
I think this line should be as shown below, currently, it's not returning a JSON result. It returns `null` when there are no doctors in the database.
```Ruby
@doctors = Doctor.all
```

[Required] I think we also need a `show` method for displaying a single doctor's profile. Currently, the show method is not implemented in the `users_controller.rb` please consider adding this method to the controller.